lib/prawn/icon/version.rb

```diff
@@ -8,6 +8,6 @@
 
 module Prawn
   class Icon
-    VERSION = '2.5.0'.freeze
+    VERSION = '2.5.1'.freeze
```

Comment:
As this adds new functionality, we should probably bump the minor version to `2.6.0`. What do you think?